docs/extensions.md

```diff
@@ -76,7 +76,9 @@ After adding them, click "Save", saving the configuration. After saving, log out
 
 • Next page will be the SP information, this is where you should provide the Single Sign On URL, and SP Entity ID.
 
-	- ACS URL : https://10.67.0.70:3443/api/v1/login_sso
+	- ACS URL : 
+ 		- For Cloud : https://shuffler.io/api/v1/login_sso
+		- For On-prem: https://<your-ip>:3443/api/v1/login_sso	
 	- Entity ID : shuffle-saml
```

Comment:
Did you read the email I sent? They were confused by the Entity ID. Does the Entity ID matter?

Please answer them as well.